website/static/css/custom.css

```diff
@@ -0,0 +1,210 @@
+  line-height: 24px;
+  margin-top: 4px;
+  margin-left: 20vw;
+}
+.button {
+  /* border: none; */
+  cursor: pointer;
+  display: inline-block;
+  font-weight: 700;
+  font-size: 0.875rem;
+  line-height: 1.625rem;
+  text-transform: none; 
+  height: auto;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  border-radius: 290486px;
+  padding: 10px 30px;
+}
+
+#footer {
+  padding: 0;
+  background: black;
+}
+#footer .sitemap {
+  width: 100%;
+  background: #ac1414;
+  display: block;
+  justify-content: center;
+  margin: 0 0;
+  max-width: none;
+  padding: 3em 0;
+}
+#footer .sitemap .wrapper {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+#footer .sitemap .wrapper > div {
+  flex-basis: 45%;
+  margin-right: 10px;
+}
+#footer .sitemap h5 {
+  padding-bottom: 10px;
+  font-size: .875rem;
+  line-height: .875rem;
+  font-weight: 700;
+  display: block;
+  border-bottom: 2px solid hsla(0,100%,80%,.2);
+}
+#footer .sitemap a {
+  font-size: 11px;
+  line-height: 20px;
+  color: white;
+}
+#footer .sitemap a:hover {
+  color: #c1c3c6;
+}
+
+#footer .footer-links {
+  background: black; 
+  padding: 1em 0;
+}
+#footer .footer-links .wrapper {
+  
+}
+#footer .footer-links .social-container {
+  display: flex;
+  justify-content: center;
+  margin: 10px 0 15px;
+}
+#footer .footer-links .social-container p {
+  color: white;
+  font-weight: 500;
+  font-size: .75rem;
+  line-height: 1.25rem;
+  margin-right: 20px;
+  margin-bottom: 0;
+}
+#footer .footer-links .social-container a {
+  margin-right: 10px;
+}
+#footer .footer-links .social-container a svg {
+  fill: white;
+}
+#footer .footer-links .social-container ul {
+  list-style: none;
+}
+#footer .footer-links .link-container {
+  display: flex;
+  justify-content: space-between;
+}
+#footer .footer-links .link-container a {
+  color: #fff;
+  font-weight: 500;
+  font-size: .75rem;
+  line-height: 1.25rem;
+}
+
+#footer .footer-copyright {
+  background: black;
+  padding: 0;
+}
+#footer .footer-copyright .copyright {
+  color: #f1f1f1;
+  font-size: 11px;
+}
+
+@media only screen and (min-device-width: 360px) and (max-device-width: 736px) {
+  
+}
+
+@media only screen and (min-width: 1024px) {
+  .fixedHeaderContainer {
+    height: 100px;
+    padding: 0;
+    margin-bottom: 1rem;
+  }
+  .navPusher {
+    padding-top: 120px;
+    min-height: calc(100vh - 120px);
+  }
+  .onPageNav {
+    max-height: calc(100vh - 110px);
+    top: 110px;
+  }
+  .fixedHeaderContainer .headerWrapper {
+    padding: 30px 20px;
+  }
+  .navigationSlider .slidingNav ul li a {
+    color: $textColor;
+    font-size: 14px;
+    margin: 0 15px;
+    padding: 20px 0;
+  }
+  .navigationSlider .slidingNav ul li a:hover, 
+  .navigationSlider .slidingNav ul li.siteNavItemActive a, 
+  .navigationSlider .slidingNav ul li.siteNavGroupActive a {
+    color: $primaryColor;
+    background: none;
+  }
+  .navigationSlider .slidingNav ul li:nth-last-child(2) a {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing vendor-prefixed CSS gradients for Firefox 3.6+, Webkit (Safari 5+, Chrome), Opera 11.1+.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4766501)